src/user/services/user.service.ts

```diff
@@ -10,17 +10,16 @@ import { plainToClass } from 'class-transformer';
 
 @Injectable()
 export class UserService {
-  constructor(private userRepository: UserRepository) {}
+  constructor(private repository: UserRepository) {}
```

Comment:
Personally like the naming just `repository` over `userRepository`. But sometimes one service needs multiple repositories. For example in BlogService might need both `blogRepository` and `authorRepository`. To cover situations like this we will need full naming.

Reply:
@asad-mlbd 
Thanks for the review.
The convention that we have followed so far on the client project is:
1. A service only imports a single repository.
2. Any other entity that needs to be updated, will be done by calling that corresponding service.

So for example if we want to update `author` from `blog service`, we will import `author service` for it. And the blog will be updated normally via the imported `blog repository`. :pray: